app/Components/AddForm.tsx

```diff
@@ -0,0 +1,317 @@
+"use client";
+import React, { ChangeEvent } from "react";
+import { useState } from "react";
+import { createEmployee } from "@/lib/createEmployee";
+
+const Addform = () => {
+  const [newEmployee, setNewEmployee] = useState<Employee>({
+    accountName: "",
+    _id: "",
+    bankAccountNumber: 0,
+    bankName: "",
+    dateOfBirth: "",
+    educationalLevel: "",
+    emailAddress: "",
+    emergencyPhoneNumber: "",
+    employmentRole: "",
+    employmentStartDate: "",
+    firstName: "",
+    gender: "",
+    lastName: "",
+    nextOfKinFullName: "",
+    nextOfKinPhoneNumber: "",
+    nextOfKinRelationship: "",
+    phoneNumber: "",
+    physicalAddress: "",
+  });
+  const [isHidden, setIsHidden] = useState<boolean>(true);
+  const formStyle: string = "px-1 md:p-2 border w-full";
+  async function handleAdd(e: any) {
+    e.preventDefault();
+    createEmployee(newEmployee);
+  }
+  function handleChange(event: ChangeEvent<HTMLInputElement>) {
+    const { name, type, value, checked } = event.target;
+    setNewEmployee((previous) => {
+      return { ...previous, [name]: type === "checkbox" ? checked : value };
+    });
+  }
+
+  return (
+    <div>
+      <div className="flex justify-between mx-10 my-6">
+        <p className="pt-4 text-center w-3/5">
+          {isHidden ? "Have a new Employee?" : "Click to return the form"}
```

Comment:
Is this Have or Add? Kindly ignore if not necessary.
As in "Add a new employee?"